packages/aws-cdk/README.md

```diff
@@ -559,6 +560,87 @@ This feature is currently in preview. Be aware of the following limitations:
   in the right order. The CLI will not help you import dependent resources in the right
   order, the CloudFormation deployment will fail with unresolved references.
 
+
+### `cdk migrate`
+
+⚠️**CAUTION**⚠️ 
+
+CDK migrate is currently experimental and may have breaking changes in the future. 
+We make no guarantees about the outcome or stability of the Command nor it's output.
+
+Generates a CDK application from an existing CloudFormation template in any of the CDK supported languages. 
+
+
+#### Required Arguments:
+
+(You must specify either `--from-path` or `--from-stack`)
+
+* `--stack-name <my_stack_name>` - The name for both the CDK application and stack. 
+* `--from-path <my_file_path>` - Takes the relative file path to the JSON or YAML CloudFormation template 
+* `--from-stack` - Retrieves a deployed cloudformation stack from your account with the same name as `--stack-name`.
```

Comment:
Do not organize the flag usage like this.

By starting with the flags, you're starting from the *solutions*. But users are reading this doc with a *problem* in their head. If you think of the readme as a hash table, the keys are *problems* and the values are *solutions*. Users reading this doc want to be able to do (in their head):

```ts
const solutionToMyProblem = README[myUseCase]
```

When you organize this with flags (*solutions*) at the top level, that mental operation doesn't work, because the doc's keys are the solutions, and not the problems. 

Yes, flags can be named in such a way that they are obviously what you want (eg, `--disable-rollback`), but this will never work for all flags. You have more expressiveness with a markdown heading than you do a CLI flag name, so well-named markdown headings will always be easier to parse than CLI flags.